lxd/container_delete.go

```diff
@@ -71,8 +30,14 @@ func containerDelete(d *Daemon, r *http.Request) Response {
 		return SmartError(err)
 	}
 
+	// TODO: i have added this not sure its a good idea (pcdummy)
```

Comment:
So you want to delete containers which LXD doesn't know about? I'm not sure this is a good idea.